### PR TITLE
Add example to docs for Array.member function

### PR DESCRIPTION
**Pull request by fozcodes**
Hey there!

My team just started using this lib and I saw in #265 that you're taking PRs to help
improve the docs. I personally would love some examples for functions like `member` so
I thought I'd add one.

I'll be sure to throw up more PRs of examples as I go.

Great library!

### CODE REVIEW

**Comment by gcanti** [comment]
Thank you @fozcodes. The md files are generated though, so you should modify the [source file](https://github.com/gcanti/fp-ts/blob/f6580b7975fd7ab2822e6f148173b3c4958a792d/src/Array.ts#L708).

Something like

```ts
/**
 * Test if a value is a member of an array. Takes a `Stetoid<A>` as a single
 * argument which returns the function to use to search for a value of type `A` in
 * an array of type `Array<A>`.
 *
 * @example
 * member(stetoid.stetoidString)(['thing one', 'thing two', 'cat in the hat'], 'thing two') // true
 *
 * @function
 * @since 1.3.0
 */
```

**Comment by fozcodes** [comment]
Ah! Of course. I knew there was some generation happening somewhere. Just picked the wrong place 🤦‍♂️ .

Updated.

File: src/Array.ts [hunk 1]
```diff
@@ -705,7 +705,13 @@ export const rotate = <A>(n: number, xs: Array<A>): Array<A> => {
 }
 
 /**
- * Test if a value is a member of an array
+ * Test if a value is a member of an array. Takes a `Stetoid<A>` as a single
```
Comment (raveclassic) [inline src/Array.ts]: there's a small type here: `Stetoid<A>`

Comment (gcanti) [reply src/Array.ts]: Thanks @raveclassic. Looks like there's another typo in the example. I think we can drop the prefix `setoid.` and go for

```
@example
import { setoidString } from 'fp-ts/lib/Setoid'

member(setoidString)(['thing one', 'thing two', 'cat in the hat'], 'thing two') // true
```

which also shows the required import

Comment (fozcodes) [reply src/Array.ts]: Great catch @raveclassic and good call @gcanti - that's much clearer to show the dependency.

Updated. 👍

**Comment by gcanti** [comment]
@raveclassic if it's ok for you I'm gonna merge this

**Comment by raveclassic** [comment]
@gcanti Yeah, ok

**Pull request merged**